src/numberWrapper.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * This class wraps a number to be able to pass it as a function parameter by reference.
+ */
+export default class NumberWrapper {
```

Comment:
Hmmmmmm... I'm wondering if we can avoid using this by tweaking the functions that take them and see if we can return the updated value.

Reply:
Fixed in 4729c650e4a6efdfb10aeead77c1d31a2d9219c9